lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -15,33 +19,48 @@ class _CategoryMainMenuState extends ConsumerState<CategoryMainMenu> {
     final gridTile = ref
         .read(homeViewModelProvider.notifier)
         .cocktailMenuTiles
-        .map((tile) => _buildGridTile(withName: tile.name))
+        .map((tile) => _buildGridTile(withName: tile.name, path: tile.path))
         .toList();
 
     return Center(
-        child: GridView.count(
-      primary: false,
-      padding: const EdgeInsets.all(20),
-      crossAxisSpacing: 10,
-      mainAxisSpacing: 10,
-      crossAxisCount: 2,
-      children: gridTile,
-    ));
+      child: GridView.count(
+        primary: false,
+        padding: const EdgeInsets.all(Spacing.spacingMD),
+        crossAxisSpacing: 20,
+        mainAxisSpacing: 10,
+        crossAxisCount: 2,
+        children: gridTile,
+      ),
+    );
   }
 
-  Widget _buildGridTile({required String withName}) {
+  // add required String routeName
+  Widget _buildGridTile(
+      {required String withName, required PageRouteInfo path}) {
+    void goTo() {
+      context.router.push(path);
+    }
+
     return GridTile(
-      child: Container(
-        padding: const EdgeInsets.all(8),
-        decoration: BoxDecoration(
+      child: TextButton(
+        onPressed: () {
+          goTo();
+        },
```

Comment:
Nothing wrong with this, just trying to make widgets less verbose 
```suggestion
        onPressed: () => goTo(),
```